core/src/models/Export.ts

```diff
@@ -137,7 +150,18 @@ export class Export extends Model<Export> {
   @BelongsTo(() => Profile)
   profile: Profile;
 
-  async markMostRecent() {
+  async setError(error: Error) {
+    this.errorMessage = error.toString();
+    if (error["errorLevel"]) {
+      this.errorLevel = error["errorLevel"];
+    }
+    // QUESTION: are we sure this shouldn't be complete?
```

Comment:
@evantahler maybe not for this PR, but some interesting semantics to talk about here.

Reply:
From chat: probably a new column to know that the export is "done" being retried